finetune/base_models/bert/model.py

```diff
@@ -206,6 +206,58 @@ def get_encoder(cls, config=None, **kwargs):
         else:
             return RoBERTaEncoder(**kwargs)
 
+class DocRep(_BaseBert):
+    encoder = RoBERTaEncoderV2
+    featurizer = bert_featurizer
+    settings = {
```

Comment:
it seems like most of these settings overlap with other models. Can we take advantage of inheritance or at least separate out the configs that are different for this base model?